libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -57,9 +58,16 @@ const AP_Param::GroupInfo AP_OAPathPlanner::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("MARGIN_MAX", 3, AP_OAPathPlanner, _margin_max, OA_MARGIN_MAX_DEFAULT),
 
+    // @Param: RESET_ORIGIN
+    // @DisplayName: Reset way point origin to current location while recovering from Object Avoidance
+    // @Description: Enabling this would reset the origin of the waypoint to the present location while recovering from Object Avoidance. Disabling it will make the vehicle return to its original waypoint trajectory.   
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    AP_GROUPINFO("RESET_ORIGIN", 4, AP_OAPathPlanner, _wp_reset, OA_WP_RESET_DEFAULT),
```

Comment:
Maybe a AP_int16 bitmask param? Then you can find 15 other things to switch on and off too!

Reply:
I have left it for now cause it doesn't make too mcuh sense to me but I will ask others and if everyone agrees, i'll do it. Thanks a lot for the review :)

Reply:
I'd recommend the param be called _OPTIONS.